src/components/ebay-menu/test/test.browser.js

```diff
@@ -188,25 +188,6 @@ describe('given the menu is in the expanded state', () => {
         });
     });
 
-    // these two tests are failing on CI.
```

Comment:
Were these commented tests removed on purpose? I'd like to keep them around and fix at some point.

Reply:
I removed them as they were commented out a year ago. We shouldn't have commented tests. We should fix them or remove them.

Reply:
Couldn't have been a year ago. These tests are for the menu shortcut which I did a month or two back.

Reply:
I'll create an issue and copy the code into it.

Reply:
Hmmm. My editor must have been lying to me then. But, yeah, we shouldn't have commented code (tests or otherwise). Moving it to an issue is a great idea.

Reply:
https://github.com/eBay/ebayui-core/issues/721